lib/rules/index.js

```diff
@@ -15,5 +15,6 @@ module.exports = new LazyLoadingRuleMap(Object.entries({
     "no-ajax": () => require("./no-ajax"),
     "no-ajax-events": () => require("./no-ajax-events"),
     "no-moment-js": () => require("./no-moment-js"),
-    "no-date-fns": () => require("./no-date-fns")
+    "no-date-fns": () => require("./no-date-fns"),
+    "lighter-image-react": () => require("./lighter-image-react")
```

Comment:
can we please rename to ligher-image-file? So we'll be consistent across ec0lint and ec0lint-style